query/src/main/java/org/infinispan/query/impl/InfinispanQueryStatisticsInfo.java

```diff
@@ -17,13 +17,6 @@
 public class InfinispanQueryStatisticsInfo implements InfinispanQueryStatisticsInfoMBean {
 
```

Comment:
I think we can keep them. A good place to add Infinispan-specific metrics, and hopefully also clustered metrics and things related to hybrid queries?